node/cn/tracers/internal/tracers/revert_tracer.js

```diff
@@ -53,6 +63,6 @@
                 this.revertString = this.toAscii(outputHex.slice(start,end));
             }
         }
-        return this.revertString;
+        return {"callerAddress": this.callerAddress, "revertMessage": this.revertString};
```

Comment:
Is it possible for user to recognize below caes?
- there is no revert error.
- there is revert error with no revert message.